hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -178,19 +181,22 @@ protected HollowProducer(BlobStager blobStager,
             int numStatesBetweenSnapshots,
             long targetMaxTypeShardSize,
             HollowMetricsCollector<HollowProducerMetrics> metricsCollector) {
-        this(blobStager, publisher, announcer, validators, listeners, versionMinter, snapshotPublishExecutor, numStatesBetweenSnapshots, targetMaxTypeShardSize, metricsCollector, new DummyBlobStorageCleaner(), new BasicSingleProducerEnforcer());
+        this(blobStager, publisher, announcer, validators, listeners, Collections.<HollowValidationListener>emptyList(), versionMinter, snapshotPublishExecutor, numStatesBetweenSnapshots, targetMaxTypeShardSize, metricsCollector, new DummyBlobStorageCleaner(), new BasicSingleProducerEnforcer());
     }
 
     protected HollowProducer(BlobStager blobStager,
                              Publisher publisher,
                              Announcer announcer,
                              List<Validator> validators,
                              List<HollowProducerListener> listeners,
+                             List<HollowValidationListener> vallisteners,
```

Comment:
minor but might be good idea to use camel case like valListeners

Reply:
Ok. Just was a bit lazy I guess. :)

Reply:
Done.